src/util/net/util.h

```diff
@@ -0,0 +1,50 @@
+#ifndef HEADER_fd_src_util_net_util_h
+#define HEADER_fd_src_util_net_util_h
+
+#include "fd_udp.h"
+#include "fd_eth.h"
+
+typedef struct __attribute__((packed)) {
```

Comment:
Can you just still all of this in `fd_udp.h` and prefix things like they are there? Eg, this struct should be named `fd_udp_ip_eth_hdr_t`

Reply:
I am not sure this is the best idea. I would then have to tie fd_udp.h to fd_eth.h as well to pull in the header definition. This struct uses all three and having a separate file makes most sense. I also think fd_net_hdrs is a more succinct but clear name

Reply:
This one part if util code so making it very precise and a higher bar than our other changes is important. There are no `util.h` type of files in util/, everything has a purpose (take a look around). So this should similarly be named in a purposeful way.

Reply:
renamed